core/views.py

```diff
@@ -43,6 +44,12 @@ def list_todos(request):
     return render(request, "list_todos.html", {"todos": todos, "form": form})
 
 
+@authenticate
+def list_todos_json(request):
+    todos = list(Todo.objects.all().order_by("id").values())
```

Comment:
You don't need the `.all()` call:

```suggestion
    todos = list(Todo.objects.order_by("id").values())
```